### PR TITLE
Moved and styled buttons on game screen

### PR DESCRIPTION
**Pull request by sean-lewis**

### CODE REVIEW

File: static/.gitkeep [hunk 1]
```diff
@@ -1 +0,0 @@
-Dummy file so that this directory can be tracked in the vagrant-django-template git repository even when empty
```
Comment (kave) [inline static/.gitkeep:1]: looks like this commit is deleting this file. can you commit it back in? thanks

File: requirements.txt [hunk 2]
```diff
@@ -24,3 +24,4 @@ six==1.9.0
 sqlparse==0.1.14
 tablib==0.10.0
 whitenoise==1.0.6
+selenium==2.45.0
```
Comment (kave) [inline requirements.txt:27]: are you currently writing selenium tests?

File: static/.gitkeep (hunk 1 above)
Comment (sean-lewis) [inline static/.gitkeep:1]: @kave Added it back

File: requirements.txt (hunk 2 above)
Comment (sean-lewis) [inline requirements.txt:27]: @kave I am going to start writing those alongside the Python tests. Did you want to do them?

Comment (kave) [inline requirements.txt:27]: @madridsoccer5 nah was just curious it was in the requirements.txt np

**Pull request merged**